packages/graphiql/src/main.tsx

```diff
@@ -0,0 +1,188 @@
+/**
+ * UMD GraphiQL Example
+ *
+ * This is a simple example that provides a primitive query string parser on top of GraphiQL props
+ * It assumes a global umd GraphiQL, which would be provided by an index.html in the default example
+ *
+ * It is used by:
+ * - the netlify demo
+ * - end to end tests
+ * - webpack dev server
+ */
+
+import React from 'react';
```

Comment:
this file should not contain imports. this example was meant to be a CDN usage example, that tests the CDN example using the UMD without esm. this example implementaiton is not a file for users to import, I hope that's clear. the vast amount of our users use the unpkg/jsdelivr bundle, not the esm import. please see the comment at the top of this file